source/plugin/text/representations/json.rst

```diff
@@ -3,12 +3,18 @@ JSON Format
 ===========
 
 JSON is `JavaScript Object Notation <https://www.json.org/>`_, a "light-weight data-interchange format" that is "easy
-for humans to read and write" and "for machines to parse and generate". The
-`Minecraft Wiki <https://minecraft.gamepedia.com/Commands#Raw_JSON_Text>`_ details the structure of text represented
-in JSON.
+for humans to read and write" and "for machines to parse and generate". The JSON web site details the structure of 
+text represented in the JSON format. In addition, the 
+`Minecraft Wiki <https://minecraft.gamepedia.com/Commands#Raw_JSON_Text>`_ provides Minecraft-related information about 
+JSON files.
 
-For example, the text "Hello World!", formatted with the color red and an underline would have the following representation
-in JSON:
+.. tip::
+
+    The data are stored in structures called nodes. Nodes contain key-value pairs and/or nested nodes. See 
```

Comment:
The data is/are stored in structures called **trees**. Each data point on a tree is a node - the node is part of the structure, not the structure itself.